docs/source/configuration/credentials.md

```diff
@@ -0,0 +1,52 @@
+# Credentials
+
+For security reasons, we strongly recommend that you *do not* commit any credentials or other secrets to version control.
+
+Kedro is set up so that, by default, if a file inside the `conf` folder (and its subfolders) contains `credentials` in its name, it will be ignored by git.
+
+## How to load credentials
```

Comment:
These examples are like the "how to directly access configuration in code" from the Basic page, however if I understand correctly the user doesn't need to do that unless they have a reason to? Or is that different for credentials?

Maybe related to my other comment in the parameters file, I'm missing a simple example of how would I use such a credential at the top of this page.